pyproject.toml

```diff
@@ -39,8 +39,8 @@ dependencies = [
 ]
 
 [tool.hatch.envs.default.scripts]
-test = "pytest {args:tests}"
-test-cov = "coverage run -m pytest {args:tests}"
+test = "pytest {args:tests tests/integration}"
```

Comment:
We should have a separate script to run the integration test.

```
hatch run test # should run only the unit tests
hatch run integration-test # should run the integration tests
```

Reply:
And then, you can add a new step to the CloudBuild cicd pipeline: https://github.com/Kaggle/kagglehub/blob/edb23ab01373ec7a903b33eec24a76806757ecd8/tools/cicd/cloudbuild.yaml#L29-L33